CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Bump `stackable-operator` from `0.80.0` to `0.83.0` ([#316]).
```

Comment:
We normally don't document op-rs bumps in here - we only do so when there are user-facing changes

Reply:
https://github.com/stackabletech/commons-operator/pull/316/commits/b0c95fd664d11b7bc86349c57f4eabe745b113a4